tests/mock_vllm_model.py

```diff
@@ -2,11 +2,12 @@
 import os
 import sys
 import time
+from datetime import datetime
+import json
 from dataclasses import dataclass
 from typing import List
 
 import torch
-from loguru import logger
 
 sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
 from tt_metal.models.demos.t3000.llama2_70b.tt.llama_common import (
```

Comment:
should the import path be `models.demos.t3000.llama2_70b.tt.llama_common` for these tt-metal modules?

Reply:
yes, you are right. We no longer use the symlink so you are correct.  It is updated now.